_tags

```diff
@@ -31,6 +31,7 @@ true: annot, bin_annot
 # OASIS_STOP
 
 true: warn(+A-4-6-7-9-40-42-44-48)
+true: warn_error(+1..49)
```

Comment:
Doesn't this mean it will trigger for 4,6, ... ? I'm not familiar with warn error ...